src/login/pages/Login.css

```diff
@@ -0,0 +1,148 @@
+.kcBodyClass {
+    background-image: url('../../assets/health-care-team.png') !important;
+    background-size: cover !important;
+    background-position: center !important;
+    height: 100vh !important;
+    display: flex !important;
+    justify-content: center !important;
+    align-items: center !important;
+    font-family: poppins, sans-serif;
+    font-size: 14px !important;
+}
+
+.KcHeaderClass, .kcHeaderWrapperClass {
+    display: none;
+}
+
+.kcFormCardClass {
+    width: 30vw;
+    min-width: 540px;
+    height: 98%;
+    background-color: white;
+    padding: 1.5em !important;
+    box-shadow: .5em 0 1em rgba(0,0,0,0.15);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    position: absolute;
+    left: 0;
+    top: 0;
+    overflow-y: auto;
+    border: none;
+
+    .logo {
+        width: 100%;
+        margin-bottom: 4em;
+        align-self: center;
+    }
+
+    #kc-page-title, .kcFormHeaderClass, .login-pf-header {
+        display: none;
+    }
+
+    .kcInputClass {
+        border: 0.1em solid #3f4373;
+    }
+
+    .kcButtonClass {
+        width: 100%;
+        padding: 12px;
+        background-color: #3f4373;
+        color: white;
+        border: none;
+        cursor: pointer;
+        margin-top: 10px;
+        transition: background-color 0.3s ease;
+    }
+
+    .kcButtonClass:hover {
+        background-color: #31355a;
+        border-color: #2d3052;
+    }
+
+    a {
+        color: #3f4373;
+    }
+
+    .divider {
+        width: 100%;
+        height: 1px;
+        background-color: #ddd;
+        margin: 20px 0;
+    }
+
+    .cookies-button {
```

Comment:
Please add support for the below UI elements. Make sure we add the hyper link to `Terms and Conditions` and `Privacy Notice`
![Screenshot 2024-12-17 at 3 56 45 PM](https://github.com/user-attachments/assets/a8b92368-b45f-481c-be69-62154e89a03d)

Reply:
Should we use the links from the previous [terms and conditions](https://learning.faimer.org/admin/tool/policy/view.php?versionid=4&returnurl=https%3A%2F%2Flearning.faimer.org%2Flogin%2Findex.php), [policy notice](https://learning.faimer.org/admin/tool/policy/view.php?versionid=3&returnurl=https%3A%2F%2Flearning.faimer.org%2Flogin%2Findex.php) or use a different implementation like develop a full-screen modals to display the details of the policies?

Reply:
![image](https://github.com/user-attachments/assets/e0f956ba-b59e-4b4c-add1-5a71db7fe560)